holoviews/core/ndmapping.py

```diff
@@ -49,11 +49,11 @@ def __init__(self, enabled):
         self.enabled = enabled
 
     def __enter__(self):
```

Comment:
The docstring of this context_manager should be updated. As now ``sort=False`` is valid, it should just say it disables sorting regardless of whether the NdMapping has ``sort=True`` or ``sort=False``. 

I also think the line 'Should only be used if values are guaranteed to be sorted before or after the operation is performed.' should just say something else - maybe just that the initial ordering (whatever it is) should be preserved?